wait/member.go

```diff
@@ -54,18 +54,24 @@ func UntilUserAccountHasSpec(expected toolchainv1alpha1.UserAccountSpec) UserAcc
 
 // UntilUserAccountMatchesMur returns a `UserAccountWaitCriterion` which checks that the given
 // MasterUserRecordSpec and UserAccountSpecEmbedded are the expected specs
```

Comment:
The description needs to be updated too.

Reply:
updated in 52a7d96